frontend/src/pages/modelServing/screens/global/EmptyModelServing.tsx

```diff
@@ -21,32 +22,36 @@ const EmptyModelServing: React.FC = () => {
 
   if (servingRuntimes.length === 0) {
     return (
-      <EmptyState variant={EmptyStateVariant.small}>
-        <EmptyStateIcon icon={WrenchIcon} />
-        <Title headingLevel="h2" size="lg">
-          No deployed models yet
-        </Title>
+      <EmptyState variant={EmptyStateVariant.sm}>
+        <EmptyStateHeader
+          titleText="No deployed models yet"
+          icon={<EmptyStateIcon icon={WrenchIcon} />}
+          headingLevel="h2"
+        />
         <EmptyStateBody>
           To get started, deploy a model from the <strong>Models and model servers</strong> section
           of a project.
         </EmptyStateBody>
-        <EmptyStateSecondaryActions>
+        <EmptyStateActions>
```

Comment:
The top margin is missing after the change, it's because PF5 removed the `EmptyStateSecondaryActions`. To bring the margin back, we should wrap `EmptyStateActions` inside `EmptyStateFooter`.

<img width="1723" alt="Screenshot 2023-11-03 at 9 57 38 AM" src="https://github.com/opendatahub-io/odh-dashboard/assets/37624318/5bc96fa8-c1fb-4239-9955-049b7b9b84c5">
<img width="1723" alt="Screenshot 2023-11-03 at 9 59 16 AM" src="https://github.com/opendatahub-io/odh-dashboard/assets/37624318/ba4b9723-57d2-4273-b22e-fe3c47addd44">

Reply:
Nice catch @DaoDaoNoCode , thank you - updated [here](https://github.com/opendatahub-io/odh-dashboard/pull/2179/files#diff-052913ac155d38fdfcda2a7b84fe25a780197d66f8c9414e4141b818b32a9873R50)